src/App.jsx

```diff
@@ -14,70 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
```

Comment:
It's better to have matching names, it can make your code easier to understand.
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```